core/trino-main/src/main/java/io/trino/execution/CreateTableTask.java

```diff
@@ -213,7 +216,14 @@ else if (element instanceof LikeClause) {
                         throw semanticException(NOT_SUPPORTED, statement, "Only one LIKE clause can specify INCLUDING PROPERTIES");
                     }
                     includingProperties = true;
-                    inheritedProperties = likeTableMetadata.getMetadata().getProperties();
+                    Collection<PropertyMetadata<?>> allProperties = tablePropertyManager.getAllProperties(catalogName);
```

Comment:
`allProperties` -> `connectorTableProperties` (or a better name suited for this context)